phys2bids/phys2bids.py

```diff
@@ -191,8 +191,11 @@ def phys2bids(filename, info=False, indir='.', outdir='.', heur_file=None,
     # Check options to make them internally coherent pt. II
     # #!# This can probably be done while parsing?
     indir = utils.check_input_dir(indir)
-    if chtrig < 1:
-        raise Exception('Wrong trigger channel. Channel indexing starts with 1!')
+    if chtrig <= 0:
+        raise Exception('Wrong trigger channel. Channel indexing cannot be negative!')
+    elif chtrig == 0:
+        LGR.warning('No trigger channel specified. phys2bids will run an automatic search based '
```

Comment:
We should add a test that checks for this warning.

Reply:
You mean for the negative warning?

Reply:
What do you mean by negative warning?

I didn't see a test checking that this warning message is shown when `chtrig==0` but I might've missed it.

Reply:
```
    # test when trigger is 0 and that the trigger channel is recognized by name:
    test_chtrig = 0
    phys_in = po.BlueprintInput(test_timeseries, test_freq, test_chn_name,
                                test_units, test_chtrig)
    assert phys_in.trigger_idx == 1
```
Look in ``test_physio_obj``

Reply:
I see, thanks!